interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/customize-appearance.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Customizing Appearance
+---
+# Customizing Appearance
+
+The topics in this section describe how to maintain the report elements' appearance:
```

Comment:
Why 'maintain'?

Reply:
I think it's better to say
The topics in this section describe how to customize the report elements' appearance:
Ok?

Reply:
Yes